libs/components/tiles/src/lib/modules/tiles/tile-dashboard/tile-dashboard.component.ts

```diff
@@ -32,11 +32,14 @@ import { SkyTileDashboardService } from './tile-dashboard.service';
   styleUrls: ['./tile-dashboard.component.scss'],
   templateUrl: './tile-dashboard.component.html',
   providers: [SkyTileDashboardService],
+  host: {
```

Comment:
Tile dashboard columns didn't reliably switch from multi-column to single-column inside a page component when using `@HostBinding` on a property, presumably because of `OnPush` change detection being used somewhere in the chain. Using a signal fixed that issue, and the new visual tests for the page component verify this so if it breaks again, those tests should fail.